libs/datamodel/core/tests/attributes/ignore_positive.rs

```diff
@@ -0,0 +1,306 @@
+use crate::common::*;
+
+#[test]
+fn allow_ignore_on_valid_model() {
+    let dml = r#"
+    model ModelId {
+      a String @id
+      
+      @@ignore
+    }
+    
+    model ModelUnique {
+      a String @unique
+      
+      @@ignore
+    }
+    
+    model ModelCompoundId {
+      a     String
+      int  Int
+      
+      @@id([a, int])
+      @@ignore
+    }
+    
+    model ModelCompoundUnique {
+      a     String
+      int  Int
+      
+      @@unique([a, int])
+      @@ignore
+    }
+    "#;
+
+    let datamodel = parse(dml);
+    datamodel.assert_has_model("ModelId").assert_ignored(true);
+    datamodel.assert_has_model("ModelUnique").assert_ignored(true);
+    datamodel.assert_has_model("ModelCompoundId").assert_ignored(true);
+    datamodel.assert_has_model("ModelCompoundUnique").assert_ignored(true);
+}
+
+#[test]
+fn allow_ignore_on_invalid_models() {
+    let dml = r#"
+    model ModelNoFields {
+     
+      @@ignore
+    }
+    
+    model ModelNoId {
+      text String 
+     
+      @@ignore
+    }
+
+    /// this does not work yet
```

Comment:
is this comment still relevant? I think it works now

Reply:
true, I made it work